src/model-helpers/Document.js

```diff
@@ -0,0 +1,168 @@
+ */
+export const signDocument = async (document, session, containerUrl) => {
+  const signingKey = await getUserSigningKey(session);
+  const signatureDataset = await signDocumentTtlFile(signingKey, document, session, containerUrl);
+  await saveSolidDatasetAt(`${containerUrl}signature.ttl`, signatureDataset, {
+    fetch: session.fetch
+  });
+};
+
+/**
+ * Helper Function that returns Driver's License ttl file based off of image passed
+ *
+ * @function createDriversLicenseTtlFile
+ * @memberof utils
+ * @function createDriversLicenseTtlFile
+ * @param {thing} thing - the thing to add info too
+ * @param {fileObjectType} file - Object containing information about file
+ * @returns {Promise<ThingLocal>} TTL file Thing - Processes a barcode using zxing
+ * and returns a new TTL file Thing
+ */
+const addDriversLicenseInfo = async (thing, file) => {
+  try {
+    const dlData = await getDriversLicenseData(file);
+    return thing
+      .addStringNoLocale(RDF_PREDICATES.additionalType, dlData.DCA)
+      .addStringNoLocale(RDF_PREDICATES.conditionsOfAccess, dlData.DCB)
+      .addDate(RDF_PREDICATES.expires, new Date(`${formattedDate(dlData.DBA)}`))
+      .addStringNoLocale(RDF_PREDICATES.givenName, dlData.DCS)
+      .addStringNoLocale(RDF_PREDICATES.alternateName, dlData.DAC)
+      .addStringNoLocale(RDF_PREDICATES.familyName, dlData.DAD)
+      .addDate(RDF_PREDICATES.dateIssued, new Date(`${formattedDate(dlData.DBD)}`))
+      .addDate(RDF_PREDICATES.dateOfBirth, new Date(`${formattedDate(dlData.DBB)}`))
+      .addStringNoLocale(RDF_PREDICATES.gender, dlData.DBC)
+      .addStringNoLocale(RDF_PREDICATES.Eye, dlData.DAY)
+      .addInteger(RDF_PREDICATES.height, Number(dlData.DAU))
+      .addStringNoLocale(RDF_PREDICATES.streetAddress, dlData.DAG)
+      .addStringNoLocale(RDF_PREDICATES.City, dlData.DAI)
+      .addStringNoLocale(RDF_PREDICATES.State, dlData.DAJ)
+      .addStringNoLocale(RDF_PREDICATES.postalCode, dlData.DAK)
+      .addStringNoLocale(RDF_PREDICATES.identifier, dlData.DAQ)
+      .addStringNoLocale(RDF_PREDICATES.identifier, dlData.DCF)
+      .addStringNoLocale(RDF_PREDICATES.Country, dlData.DCG)
+      .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDE)
+      .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDF)
+      .addStringNoLocale(RDF_PREDICATES.additionalName, dlData.DDG);
+  } catch (e) {
+    return thing;
+  }
+};
+
+/**
+ * Function that generates checksum for uploaded file
+ *
+ * @memberof Document
+ * @function createFileChecksum
+ * @param {fileObjectType} file - Object containing information about file
+ * from form submission (see {@link fileObjectType})
+ * @returns {Promise<WordArray>} Promise - Generates checksum for uploaded file
+ * using the SHA256 algorithm
+ */
+const createFileChecksum = async (file) => {
+  const text = await file.text(); // only hash the first megabyte
+  return sha256(text);
+};
+
+/**
```

Comment:
Think the JSDoc will do for now. Can do some clean-up to make it more detailed in another PR.